packages/cardano-services-client/src/TxSubmitProvider/TxSubmitApiProvider.ts

```diff
@@ -25,6 +40,7 @@ export class TxSubmitApiProvider implements TxSubmitProvider {
       this.#logger.debug(`Submitting tx ${txId} ...`);
 
       await this.#axios({
+        adapter: this.#adapter,
```

Comment:
A test like `it('supports custom axios adapter')` would be nice. Although I don't see any tests for this provider, in which case it's not needed and it's just tech debt.

Reply:
@iccicci @mirceahasegan was this merged from conway-era and we forgot to write tests? Don't remember why we added the provider :thinking: